docs/Makefile

```diff
@@ -0,0 +1,29 @@
+# Makefile for php-src/docs
+# Copyright (c) The PHP Group
+
+# If people set these on the make command line, use 'em
+
+SPHINXBUILD ?= sphinx-build
+GH_PAGES_URL ?= https://php.github.io/php-src/
+
+SOURCEDIR = source
+BUILDDIR = build
+
+rwildcard = $(foreach d,$(wildcard $(1:=/*)),$(call rwildcard,$d,$2) $(filter $(subst *,%,$2),$d))
+FILES = $(call rwildcard,$(SOURCEDIR),*.rst)
+
+all : html
+
+.PHONY : clean html
+
+clean :
+	rm -rf -- $(wildcard $(SOURCEDIR)/.~ $(BUILDDIR))
+
+html : $(SOURCEDIR)/.~
+	$(SPHINXBUILD) -M $@ $(SOURCEDIR) $(BUILDDIR)
+	@printf 'Browse the php-src docs \e]8;;%s\e\\%s\e]8;;\e\\ (or \e]8;;%s\e\\%s\e]8;;\e\\)\n' \
+		"file://$(abspath $(BUILDDIR))/$@/index.$@" "local" "$(GH_PAGES_URL)" "online"
```

Comment:
Why do we spell out the online version? That makes it sound like it was deployed.

Reply:
Found it helpful to have the reference at hand. What would it make sound better to you? @iluuu1994

Reply:
Removed; deferred for a later improvement if any.